wETH Interest Rate Curve - Ethereum v2

defijesus opened the pull request.

defijesus committed:
ready for internal review

efecarranza commented on src/ProposalPayload.sol:
```diff
@@ -1,17 +1,39 @@
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x9ae28e9c82c5fc0d24cf1df788094e959d99f906d11b89e455a60ee16b071d6f
  */
 contract ProposalPayload {
+    address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```
WETH should come from address-book.

defijesus replied:
cant find WETH on the address book, maybe I'm missing something

efecarranza commented on src/ProposalPayload.sol:
```diff
@@ -1,17 +1,39 @@
+     defijesus.eth
+*/
+
 pragma solidity ^0.8.15;
```
Fixed version

efecarranza commented on src/test/ProposalPayloadE2E.t.sol:
```diff
@@ -30,7 +30,10 @@ contract ProposalPayloadE2ETest is Test {
     function testExecute() public {
         // Pre-execution assertations
-
+        // I'm having issues what to even test here. if the deployment of the new interest rate strategy is correct,
```
Test the new strategy address
Test the new strategy values like we did for wMATIC

defijesus replied:
do you know any easy way to test those as we did for v3?

couldn't find the equivalent for v2

efecarranza reviewed (commented):
Add README.md with changes and interest rate strategy values

defijesus committed:
fix payload version
improve testing
fix utils/AaveV2Helpers.sol to use latest AaveAddressBook (it is hardcoded for AaveEthereumV2)
